en.json

```diff
@@ -0,0 +1,535 @@
+{

```

Comment:
At first look, I don't see any major difference in this file. Do you know why all of it is marked as modified?
...
Had to retrieve the file and compare it by hand to see the changes you made.
Looks good to me